draft-ietf-dnsop-dns-capture-format-00.md

```diff
@@ -1006,7 +1002,7 @@ draft-dickinson-dnsop-dns-capture-format-00
 # CDDL
 
     ; CDDL specification of the file format for C-DNS, 
-    ; which describes a collection of DNS Query/Response pairs.
+    ; which describes a collection of DNS Q/R data items.
```

Comment:
Hum. Actually, it strikes me that it does more than that. Maybe we make this more generic and say it describes a collection of DNS messages and associated traffic meta data? Or make it more detailed....